### PR TITLE
formated code

### PR DESCRIPTION
**Pull request by mattyx14**
remove blank spaces and formated code style

remove: "**;**"
https://user-images.githubusercontent.com/2807739/68533548-b8421800-02ef-11ea-9ad9-464bd565de04.png

### CODE REVIEW

**Comment by majestyotbr** [comment]
Please, explain better your pull request.

File: src/protocolgame.cpp [hunk 1]
```diff
@@ -144,7 +144,7 @@ void ProtocolGame::login(const std::string& name, uint32_t accountId, OperatingS
 		if (!IOLoginData::loadPlayerPreyData(player)) {
 			std::cout << "Prey data could not be loaded" << std::endl;
 			return;
-		};
+		}
```
Comment (mattyx14) [inline src/protocolgame.cpp]: error solved

**Review by mattyx14 — COMMENTED** [review]
error here
image

**Comment by mattyx14** [comment]
update main comment

**Comment by dudantas** [comment]
your fix is only ;? are you kidding me?

**Comment by mattyx14** [comment]
it is just a ";" it is over there 
but then the compilation does not affect anything

**Comment by dudantas** [comment]
the removal of ";" It changes nothing, simply aesthetic.
anyway, thanks.
There are a lot of codes in need of revision and several issues to be fixed, if you are interested in helping, we would be grateful.

**Pull request closed without merging**